java_training/jan20/Employee_mgmt_sql

```diff
@@ -0,0 +1,84 @@
+SET @OLD_UNIQUE_CHECKS=@@UNIQUE_CHECKS, UNIQUE_CHECKS=0;
+SET @OLD_FOREIGN_KEY_CHECKS=@@FOREIGN_KEY_CHECKS, FOREIGN_KEY_CHECKS=0;
+SET @OLD_SQL_MODE=@@SQL_MODE, SQL_MODE='TRADITIONAL,ALLOW_INVALID_DATES';
+
+CREATE SCHEMA IF NOT EXISTS `try1` DEFAULT CHARACTER SET utf8 COLLATE utf8_general_ci ;
+USE `try1` ;
+
+-- -----------------------------------------------------
+-- Table `try1`.`Designation`
+-- -----------------------------------------------------
+CREATE TABLE IF NOT EXISTS `try1`.`Designation` (
+  `ID` INT NOT NULL,
```

Comment:
The standard in MySQL is that you name your column with lower case. Eg. `id`, `employee_id`